doc/operation-and-maintenance/Mongoose-metrics.md

```diff
@@ -64,14 +64,12 @@ A histogram collects values over a sliding window of 60s and exposes the followi
 
 ### Hook metrics
 
+There are more hooks metrics than enumerated in this table, because they are automatically created for every new hook.
```

Comment:
There are more hook metrics than what is listed in this table, because they are automatically created for every new hook.